.orchestra/config/components/revng-vscode-web.yml

```diff
@@ -0,0 +1,71 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_vscode_web_args():
+repository: vscode
+binary_archives: private
```

Comment:
I'd be way happier if we didn't have multi-component repositories. I'd use those as a last resource.
We're forcing whoever wants to change this small component to clone all of vscode.
Can we pull into orchestra stuff that we need from the source directory? Would that make sense?

Reply:
Moved some files around, now `revng-vscode-web` does not need the vscode repo

Reply:
Yay!